bookstore/client/views.py

```diff
@@ -46,6 +46,17 @@ def check(email):
 app.config["MAIL_USERNAME"] = "bookly1120"
 mail = Mail(app)
 
+def send_warning_email(email):
+    import smtplib
+    con = smtplib.SMTP("smtp.gmail.com",587)
+    con.ehlo()
+    con.starttls()
+    admin_email = "admin_email"
+    admin_password = "admin_password"
+    con.login(admin_email,admin_password)
```

Comment:
@Akshay6890 don't forgot to insert a email from which email will send .

After that you can check this PR just try login in to your account with wrong password more than 5 times

Reply:
> @Akshay6890 don't forgot to insert a email from which email will send .
> 
> After that you can check this PR just try login in to your account with wrong password more than 5 times

that email variable will store the email right?

Reply:
no

Reply:
this variable is for the email from which the email has to be send @Akshay6890

Reply:
> no

so we have to hard code the email?

Reply:
in short .
 email and password of admin or owner of website @Akshay6890

Reply:
> in short .
> email and password of admin or owner of website @Akshay6890

yea, I get it, my bad 😅

Reply:
is there any variable for email and  password @Akshay6890

Reply:
> is there any variable for email and password @Akshay6890

we used to create sessions right?

Reply:
> is there any variable for email and password @Akshay6890

we assigned those values when user logs in

Reply:
> is there any variable for email and password @Akshay6890

of course only email